src/components/inherited-map/components/event/event-comments.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from "react";
 
 import { Colors } from "../../../../styles/colors";
 
-const EventComments = () => {
+export const EventComments: React.VoidFunctionComponent = () => {
```

Comment:
Bonus: I added `React.VoidFunctionComponent` in a few places to slightly improve type coverage